Source/WebKit/UIProcess/GPU/GPUProcessProxy.cpp

```diff
@@ -594,6 +594,10 @@ void GPUProcessProxy::didFinishLaunching(ProcessLauncher* launcher, IPC::Connect
     
 #if USE(RUNNINGBOARD)
     m_throttler.didConnectToProcess(*this);
+#if USE(EXTENSIONKIT)
+    if (launcher)
+        launcher->releaseLaunchGrant();
```

Comment:
It is "okayish" to do this in subclasses for now. However, I think it's be a lot less error-prone if we notified the process throttler and released the launch grant in the base class. I think we should consider refactoring in a follow-up at least.

This would have avoided you introducing a bug. You forgot to release the grant in `ModelProcessProxy::didFinishLaunching()`.

Reply:
Ah, good catch, Chris! I also think refactoring this is the best path forward. I can look into that in a follow up. The motivation for not doing the refactoring now was to minimize the risk of regressions.

Thanks for reviewing!

Reply:
Maybe we can add a FIXME comment at least.